.eslintrc

```diff
@@ -8,6 +8,29 @@
     "max-len": [2, 120],
     "react/jsx-filename-extension": 0,
     "import/no-extraneous-dependencies": 0,
-    "react/prefer-stateless-function": [2, { "ignorePureComponents": true }]
+    "react/prefer-stateless-function": [2, { "ignorePureComponents": true }],
+
+    // We've decided we don't want this for now as it makes writing JSX more painful.
+    "function-paren-newline": 0,
```

Comment:
Can we put a link to the issue here:

https://github.com/airbnb/javascript/issues/1584#issuecomment-335667272

Reply:
Done